src/components/ConnectionInterface.vue

```diff
@@ -140,6 +140,19 @@
         }
         this.$noty.success(`${config.name} deleted`)
       },
+      async duplicate(config) {
+        const duplicateConfig = Object.assign( Object.create( Object.getPrototypeOf(config)), config);
+        duplicateConfig.id = null
+        duplicateConfig.name += ' (copy)'
```

Comment:
Due to our text trimming, can we do this the other way around?

"Copy of Foo Bar"